app/Http/Controllers/Frontend/Admin/EventController.php

```diff
@@ -20,6 +20,19 @@
 
 class EventController extends Controller
 {
+
+    private const VALIDATOR_RULES = [
+        'name'                 => ['required', 'max:255'],
+        'hashtag'              => ['required', 'max:30'],
+        'host'                 => ['required', 'max:255'],
+        'url'                  => ['nullable', 'url'],
+        'nearest_station_name' => ['required', 'max:255'],
+        'begin'                => ['required', 'date'],
+        'end'                  => ['required', 'date'],
+        'event_start'          => ['required', 'date', 'after_or_equal:begin'],
+        'event_end'            => ['required', 'date', 'before_or_equal:end'],
```

Comment:
Hmm, i don't see a reason why we need to make all of this required.

Imo we just need `event_start` and `event_end` required, begin and end is optionally. Most of events doesn't need another date for showing them.